### PR TITLE
Admin can make admins

### LINKED ISSUE #257: handle admin levels here too
we take admin or not value from oneauth

but we need a way to manage admin flag on users here at boss as well. so someone who is not admin on oneauth can be marked as admin here too.

### PR DESCRIPTION
**Pull request by prathambatra**
Signed-off-by: prathambatra <batrapratham999@gmail.com>
fixes #257 
made a new page at /admin which only admins can open and edit. I haven't done styling till now. also, for failure and success admin creation should I add a flash message. please check end to end flow. @championswimmer

### CODE REVIEW

**Comment by prathambatra** [comment]
Screenshot from 2020-05-22 04-23-10
Screenshot from 2020-05-22 04-23-20

**Comment by prathambatra** [comment]
@championswimmer

**Comment by prathambatra** [comment]
@championswimmer can you please review this? I didn't see any activity from 4 days that's why writing this comment

**Comment by championswimmer** [comment]
Kinda held on the bootstrap migration, almost everything

**Comment by prathambatra** [comment]
> Kinda held on the bootstrap migration, almost everything

oh okay..

**Comment by prathambatra** [comment]
will do bootstrap changes and commit soon.

File: views/pages/admin.hbs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<div class="ui two column grid">
```
Comment (championswimmer) [inline views/pages/admin.hbs:1]: these are still semantic classes

Comment (championswimmer) [reply views/pages/admin.hbs:1]: oh sorry, you said you *will* do, i thought you said you *have* done. my bad

**Comment by prathambatra** [comment]
Screenshot from 2020-05-28 14-04-22

**Comment by prathambatra** [comment]
done..

**Comment by prathambatra** [comment]
@championswimmer any updates?

**Comment by prathambatra** [comment]
@championswimmer any updates?

**Comment by championswimmer** [comment]
Show usernames too in list of admins. Not just id.

**Comment by prathambatra** [comment]
@championswimmer please clear this, so I am adding a new column to users table i.e, 'username'. so at login, we will store both id and username from oneauth. so should I remove all the entries in users table so from now onwards only entries with username will get into users.

**Comment by championswimmer** [comment]
Why would such a case happen? User ids are unique, how can two entries get created for same user? 

Make a username column. If username is not there it'll be null. When user logs in next time in future we'll take their username and populate that column.

**Comment by championswimmer** [comment]
Do not ever make two rows with same oneauthid. That's it.

**Comment by prathambatra** [comment]
done..

**Comment by prathambatra** [comment]
@championswimmer

**Pull request merged**